package-overrides/github/aspnet/jquery-validation-unobtrusive@3.2.2.json

```diff
@@ -0,0 +1,14 @@
+{
+  "main": "jquery.validate.unobtrusive",
+  "shim": {
+    "jquery.validate.unobtrusive": {
+      "deps": [
+        "jquery-validation"
+      ],
+      "exports": "$"
+    }
+  },
+  "dependencies": {
+    "jquery-validation": "github:jzaefferer/jquery-validation"
```

Comment:
How do you specify `>=1.8` ? (see [here](https://github.com/aspnet/jquery-validation-unobtrusive/blob/master/bower.json))